src/stores/authStore2.ts

```diff
@@ -145,6 +145,27 @@ export class AuthStore {
       })
       return true
     }
+
+    // handle replace navSubMenus when login to make sure app will not auto open Phone Appli app
```

Comment:
Handle replace navSubMenus when login to make sure app will not auto open Phone Appli app